adapters/tlsio_openssl.c

```diff
@@ -150,6 +153,16 @@ static void* tlsio_openssl_CloneOption(const char* name, const void* value)
                 /*return as is*/
             }
         }
+        else if (strcmp(name, SU_OPTION_X509_CRYPTODEV_PRIVATE_KEY) == 0)
+        {
+	    result = malloc(sizeof(TLSIO_CRYPTODEV_PKEY));
+
+	    if (result != NULL) {
```

Comment:
>{ [](start = 25, length = 1)

nit: please adhere to existing style conventions in file.

Reply:
Thanks, I'll fix where I find it (is it tabs vs spaces?). Do you have some script to check it by the way?

Reply:
Deleted tabs where I could find them.